File: 競技プログラミングの鉄則/B02.py
#A~Bの間に100の約数あるか

a,b = map(int,input().split())
divisor = [1,2,4,5,10,20,25,50,100]
ans = False
  
for i in range(a, b+1):
  if i in divisor:
    ans = "Yes"
    break
  else:
    ans = "No"

print(ans)

File: 競技プログラミングの鉄則/B02.cpp
#include <bits/stdc++.h> 
using namespace std;
 
int main(){
  int a,b;
  cin >> a >> b;
  
  string ans;
  
  for (int i = a; i < b+1; i++) { //a以上b以下
    if (100 % i == 0) {
      ans = "Yes";
      break;
    }else{
      ans = "No";
    }
  }
  
  cout << ans << endl;
}
